File: code-coaches/easy/extra-terrestrials/ExtraTerrestrials.java
import java.util.Scanner;

public class ExtraTerrestrials {

    public static String reversedText(String text) {
        StringBuilder sb = new StringBuilder(text);
        sb.reverse();
        return sb.toString();
    }

	public static void main(String[] args) {
		Scanner input = new Scanner(System.in);
		String text = input.nextLine();

		System.out.println(reversedText(text));

		input.close();
	}
	
}

File: code-coaches/easy/extra-terrestrials/extra-terrestrials.cpp
#include <iostream>

std::string reverse(std::string str)
{
	std::string reversedString = "";

	for (int i = str.length() - 1; i >= 0; i--)
	{
		reversedString += str[i];
	}

	return reversedString;
}

int main()
{
	std::string text;
	getline(std::cin, text);

	std::cout << reverse(text);

	return 0;
}